tests/testthat/test_timeshift.R

```diff
@@ -11,6 +11,8 @@ if(require(mockr, quietly = TRUE, warn.conflicts = FALSE)) {
 
   test_that("chunks handle timeshift", {
 
+    skip_on_travis()
```

Comment:
If we do this, then we will have to instruct reviewers to run tests manually before approving PRs, since the CI will no longer be running the full suite of tests.